scripts/init

```diff
@@ -32,4 +32,35 @@ devenv() {
   fi
 }
 
+_devenv() {
+  . ${DEVENVROOT}/scripts/func.d/bash_utils
+
+  local cur prev opts base
+	COMPREPLY=()
+	cur="${COMP_WORDS[COMP_CWORD]}"
+	base="${COMP_WORDS[0]}"
+	cmd="${COMP_WORDS[1]}"
+
+  case "${cmd}" in
+    use|del)
+      [[ $COMP_CWORD > 2 ]] && return 1
+      COMPREPLY=( $(compgen -W "$(get_list flavor)" -- ${cur}) )
+			return 0
+      ;;
+    build)
+      [[ $COMP_CWORD > 2 ]] && return 1
+      COMPREPLY=( $(compgen -W "$(get_list build)" -- ${cur}) )
+			return 0
```

Comment:
All statements should with the same indentation level to the same block of code.

Reply:
@j8xixo12 you have tab/space mixed.  Please fix.